notsobot/notsobot.py

```diff
@@ -1373,6 +1429,10 @@ def do_haah(self, b):
             h2.flop()
             h1.save(file=f)
             h2.save(file=f2)
+            img.close()
```

Comment:
This is unnecessary since `img` was created by the context manager which closes it automatically.